### PR TITLE
Portfolio: setup and mobile first - First Milestone

### PR DESCRIPTION
**Pull request by marcoshdezcam**
### First Milestone

- Toolbar & Headline sections. 
- Layout created with Flexbox. 
- Button interactions.
- Using template 3 from the provided [Figma designs](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 

### 👤 Author
@marcoshdezcam

### 🔴 Preview
[Live version](https://marcoshdezcam.github.io/setup-mobile-first/)

### 🔧 Built With
- HTML
- CSS
- Flexbox
- Bootstrap

Thank you for taking the time to review our code! 💐

### CODE REVIEW

**Comment by PinkMoon25** [comment]
Hi @marcoshdezcam , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
## Invalid Code Review Request

You have submitted a project, that does not passes all linters check please fix the below issue and resubmit 
```
style.css
 22:16  ✖  Unexpected unit   length-zero-no-unit
 22:20  ✖  Unexpected unit   length-zero-no-unit
```

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by marcoshdezcam** [comment]
> 
Thank you! @PinkMoon25
I've submitted a fix. All checks have passed now.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" type="text/css" href="style.css" />
+  </head>
+  <body>
+    <header class="d-flex container-md p-4 justify-content-between">
+      <span>Welcome</span>
```
Comment (pinkmoon25) [inline index.html]: Please treat this as a logo and put it inside a link` <a> `tag, This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.
`<a href="#home"><span>Welcome</span></a>`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+          <button type="button" class="button">
+            <img src="icons/medium.svg" alt="Medium" />
+          </button>
+          <button type="button" class="button">
+            <img src="icons/github.svg" alt="Github" />
+          </button>
+          <button type="button" class="button">
+            <img src="icons/angellist.svg" alt="Angellist" />
+          </button>
+        </div>
```
Comment (pinkmoon25) [inline index.html]: Please fix the social media icons as according to the requiremetns we are supposed to have :
The list of icons uses a `ul` tag to group them, and `li` tag per icon; each icon has a link (`a` tag).

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.
for example :point_down: 
```
<ul>
    <li> <a href="">social media icon</i></a> </li>
    <li> <a href="">social media icon</a> </li>
    <li> <a href="">social media icon</a> </li>
    <li> <a href="">social media icon</a> </li>
    <li> <a href="">social media icon</a> </li>
</ul>
```

**Review by pinkmoon25 — CHANGES_REQUESTED** [review]
Hi @marcoshdezcam , :wave:  

Good job so far! This has to be the best one I have seen so far, there are very few things left as per the requirements, once you fix them you are good to go
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
Linters passing :heavy_check_mark: 
Correct github flow :heavy_check_mark: 
Professional Readme :heavy_check_mark: 
Follows figma template :100: 
Flex ninnja :ninja: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (marcoshdezcam) [reply index.html]: @PinkMoon25 Thank you for the feedback. I've submitted the proper fixes.

**Review by pinkmoon25 — APPROVED** [review]
Hi @marcoshdezcam , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Linters passing :heavy_check_mark: 
- Correct github flow  :heavy_check_mark: 
- Professional Readme :heavy_check_mark: 
- Follows figma template :100: 
- Flex ninnja :ninja: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by marcoshdezcam** [comment]
Project approved!

**Review by TadesseDev — COMMENTED** [review]
### Hi @marcoshdezcam 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
68747470733a2f2f6669636f666f72756d732e6d796669636f2e636f6d2f74352f696d6167652f736572766572706167652f696d6167652d69642f343932363069463839303835363833444139334135422f696d6167652d73697

## Highlights
 - [ ]  Correct GitHub Flow 👍🏾
 - [ ]  Good commit history 🥇
 - [ ]  Good PR title/description 💯
 - [ ]  All linters passed 🥇
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**